astropy/utils/iers/iers.py

```diff
@@ -40,13 +40,13 @@
 
 # IERS-A default file name, URL, and ReadMe with content description
 IERS_A_FILE = 'finals2000A.all'
-IERS_A_URL = 'https://maia.usno.navy.mil/ser7/finals2000A.all'
-IERS_A_URL_MIRROR = 'https://datacenter.iers.org/data/9/finals2000A.all'
+IERS_A_URL = 'https://datacenter.iers.org/data/9/finals2000A.all'
+IERS_A_URL_MIRROR = 'https://maia.usno.navy.mil/ser7/finals2000A.all'
 IERS_A_README = get_pkg_data_filename('data/ReadMe.finals2000A')
 
 # IERS-B default file name, URL, and ReadMe with content description
-IERS_B_FILE = get_pkg_data_filename('data/eopc04_IAU2000.62-now')
-IERS_B_URL = 'http://hpiers.obspm.fr/iers/eop/eopc04/eopc04_IAU2000.62-now'
+IERS_B_FILE = get_pkg_data_filename('data/eopc04_14_IAU2000.62-now')
+IERS_B_URL = 'https://datacenter.iers.org/products/eop/long-term/c04_14/iau2000/eopc04_14_IAU2000.62-now'
```

Comment:
Is this number `_14_` going to change every time they update it?

Reply:
it's in the `c04_14` section under `long-term` so i don't think so. it's at least been stable since 2017, fwiw.